pkg/cmd/step/expose/step_expose.go

```diff
@@ -107,7 +108,8 @@ func (o *StepExposeOptions) Run() error {
 	if err != nil {
 		return errors.Wrapf(err, "failed to load requirements YAML")
 	}
-	if requirements.Cluster.Namespace == "" {
+
+	if o.Namespace != "" {
```

Comment:
Trying to understand how this is related to the problem of reading the requirements file? Is this trying to fix yet another issue?

Reply:
This is my mistake. I had a problem with a unit test, which was fixed by adding `jx-requirements.yml` in `test_data` folder. I will remove it in the next commit.

Reply:
thanks, I was a bit baffled about this part.